giskard/llm/generators/base.py

```diff
@@ -31,10 +32,12 @@ def __init__(
         llm_temperature: Optional[float] = None,
         llm_client: LLMClient = None,
         prompt: Optional[str] = None,
+        languages_list: list[str] = ['en'],
```

Comment:
Better not to set a default argument with a mutable, see for example [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). In this case, the usual way is to set it as optional with a default to `None` and then use `self.languages = languages or ["en"]` (or similar).

Also, naming: no need to specify the type in the variable name. `languages: list[str]` is already very explicit.

Reply:
Maybe if there are no languages we should just avoid including the extra line in the prompt, no?